### PR TITLE
SUBMARINE-1049. Static registered model information page

### PR DESCRIPTION
**Pull request by FYRichie**
### What is this PR for?
Static registered model information page

### What type of PR is it?
Feature

### Todos
* [ ] - Task

### What is the Jira issue?

https://issues.apache.org/jira/browse/SUBMARINE-1049

### Questions:
* Do the license files need updating? No
* Are there breaking changes for older versions? No
* Does this need new documentation? No

### CODE REVIEW

**Comment by FYRichie** [comment]
@KUAN-HSUN-LI can you help me review my code? Thx.

**Comment by KUAN-HSUN-LI** [comment]
@FYRichie Thanks for your contribution.

The pull request title must follow the following format:

SUBMARINE-${jira_number}. ${jira_summary}

File: submarine-workbench/workbench-web/src/app/pages/workbench/model/registered-model/registered-model.component.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+<div style="margin: 15px; padding: 15px; background-color: white;">
+    <div><h1 style="font-weight: lighter;">${Registered Model Name}</h1></div>
+    <div>
+        <div style="margin: 5px 0px 5px 0px; font-size: 17px; font-family: Arial;">Description: tdis is a description.</div>
+        <div style="margin-bottom: 5px; font-size: 17px; font-family: Arial;">
+            <div style="display: inline-block;">Tags:</div>
+            <div style="display: inline-block; background-color: #53c653; margin: 3px; padding: 3px;">image</div>
+        </div>
+        <div style="margin-bottom: 5px; font-size: 17px; font-family: Arial;">Created: 2021-09-01 07:57</div>
+        <div style="margin-bottom: 5px; font-size: 17px; font-family: Arial;">Updated: 2021-09-21 03:00</div>
```
Comment (kobe860219) [inline submarine-workbench/workbench-web/src/app/pages/workbench/model/registered-model/registered-model.component.html:29]: The style of div in line 24, 28 and 29 is same, maybe you can use class css selector for simplify.

File: submarine-workbench/workbench-web/src/app/pages/workbench/model/registered-model/registered-model.component.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    </div>
+    <div style="margin-top: 20px;">
+        <div style="margin-bottom: 5px; font-size: 17px; font-family: Arial; font-weight: bold; display: inline-block;">Model versions info</div>
+        <div style="margin-left: 100px; display: inline-block;">
+            <div style="margin-bottom: 5px; font-size: 17px; font-family: Arial; display: inline-block;">Tags:</div>
+            <div style="margin-bottom: 5px; font-size: 17px; font-family: Arial; display: inline-block;">
+                <input type="text" style="height: 35px; width: 250px; border:1px solid black; font-size: 17px;" placeholder="Model Version tags...">
```
Comment (kobe860219) [inline submarine-workbench/workbench-web/src/app/pages/workbench/model/registered-model/registered-model.component.html:36]: One question, the input type is text, if the all tags total length over the 250px width, wrap may not be triggered.

**Review by kobe860219 — COMMENTED** [review]
I made some comments in html [file](submarine-workbench/workbench-web/src/app/pages/workbench/model/registered-model/registered-model.component.html).

File: submarine-workbench/workbench-web/src/app/pages/workbench/model/registered-model/registered-model.component.html (hunk 1 above)
Comment (FYRichie) [reply submarine-workbench/workbench-web/src/app/pages/workbench/model/registered-model/registered-model.component.html:29]: Thanks for the advice, I'll change it

File: submarine-workbench/workbench-web/src/app/pages/workbench/model/registered-model/registered-model.component.html (hunk 2 above)
Comment (FYRichie) [reply submarine-workbench/workbench-web/src/app/pages/workbench/model/registered-model/registered-model.component.html:36]: Are there any recommended solutions?

Comment (kobe860219) [reply submarine-workbench/workbench-web/src/app/pages/workbench/model/registered-model/registered-model.component.html:36]: Textarea type may be one good solution. It needs test.

**Comment by FYRichie** [comment]
@KUAN-HSUN-LI @kobe860219 I've changed the pr title and some code details, can you help me review my code?

File: submarine-workbench/workbench-web/src/app/pages/workbench/model/registered-model/registered-model.component.html [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under tde License is distributed on an
+  ~ "AS IS" BASIS, WItdOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, eitder express or implied.  See tde License for tde
+  ~ specific language governing permissions and limitations
+  ~ under tde License.
+  -->
+<style>
```
Comment (kobe860219) [inline submarine-workbench/workbench-web/src/app/pages/workbench/model/registered-model/registered-model.component.html:19]: CSS style here could be wrote in scss file.

File: submarine-workbench/workbench-web/src/app/pages/workbench/model/registered-model/registered-model.component.html [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+  ::placeholder {
+    font-size: 17px;
+  }
+  .info-box {
+    margin-bottom: 5px;
+    font-size: 17px;
+    font-family: Arial;
+    display: inline-block;
+  }
+</style>
```
Comment (kobe860219) [inline submarine-workbench/workbench-web/src/app/pages/workbench/model/registered-model/registered-model.component.html:40]: CSS style could be wrote in scss file.

**Comment by pingsutw** [comment]
Some tests failed, could you take a look?
https://github.com/apache/submarine/runs/4114213716?check_suite_focus=true

**Comment by FYRichie** [comment]
@KUAN-HSUN-LI Can you help me review my code?

File: submarine-workbench/workbench-web/src/app/pages/workbench/model/registered-model/registered-model.component.html [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under tde License is distributed on an
+  ~ "AS IS" BASIS, WItdOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, eitder express or implied.  See tde License for tde
+  ~ specific language governing permissions and limitations
+  ~ under tde License.
+  -->
+<div style="margin: 15px; padding: 15px; background-color: white">
```
Comment (KUAN-HSUN-LI) [inline submarine-workbench/workbench-web/src/app/pages/workbench/model/registered-model/registered-model.component.html:19]: CSS should be written in `.scss` file.

File: submarine-workbench/workbench-web/src/app/pages/workbench/model/registered-model/registered-model.component.ts [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+import { ActivatedRoute, Router } from '@angular/router';
+import { ModelInfo } from '@submarine/interfaces/model-info';
+
+@Component({
+  selector: 'app-registered-model',
+  templateUrl: './registered-model.component.html',
+  styleUrls: ['./registered-model.component.scss'],
+})
+export class RegisteredModelComponent implements OnInit {
+  name;
```
Comment (KUAN-HSUN-LI) [inline submarine-workbench/workbench-web/src/app/pages/workbench/model/registered-model/registered-model.component.ts:30]: Change the variable to `modelName`.

**Review by KUAN-HSUN-LI — CHANGES_REQUESTED** [review]
Others, LGTM.

File: submarine-workbench/workbench-web/src/app/pages/workbench/model/registered-model/registered-model.component.html (hunk 5 above)
Comment (FYRichie) [reply submarine-workbench/workbench-web/src/app/pages/workbench/model/registered-model/registered-model.component.html]: @KUAN-HSUN-LI I've finished the above suggestions, pls help me review my code, thx.

**Review by KUAN-HSUN-LI — APPROVED** [review]
Thanks for the contribution.

**Pull request closed without merging**